client/src/components/metrics/Graph.tsx

```diff
@@ -97,7 +97,12 @@ export default function Graph() {
         <BarChart margin={{top: 40, right: 0, bottom: 40, left: 50}} data={numStudentsPerDay}>
           <CartesianGrid stroke="#ccc" />
           <XAxis dataKey="day" />
-          <YAxis dataKey="students">
+          <Label
```

Comment:
the axis labels don't quite pop out in dark mode since default is black. not sure how to change with recharts but I imagine it'd be changing the style in Label here?